concepts/05 Widgets/LoadIndicator/00 Overview.md

```diff
@@ -44,6 +44,74 @@ The following code adds a simple **LoadIndicator** to your page. You can change
         // ...
     })
 
+##### Vue
+
+    <template>
+        <DxLoadIndicator
+            :visible.sync="isLoadIndicatorVisible"
+            :height="40"
+            :width="40"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxLoadIndicator } from 'devextreme-vue/load-indicator';
+
+    export default {
+        components: {
+            DxLoadIndicator
+        },
+        data() {
+            return {
+                isLoadIndicatorVisible: true
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { LoadIndicator } from 'devextreme-react/load-indicator';
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+            this.state = {
+                isLoadIndicatorVisible: true
+            };
+            this.handleValueChange = this.handleValueChange.bind(this);
+        }
+
+        handleValueChange(e) {
+            const previousValue = e.previousValue;
+            const newValue = e.value;
+
+            this.setState({
+                isLoadIndicatorVisible: newValue
+            });
+        }
+
+        render() {
+            return (
+                <LoadIndicator
+                    value={this.state.isLoadIndicatorVisible}
+                    onValueChanged={this.handleValueChange}
```

Comment:
The LoadIndicator doesn't have a `value` or `onValueChanged` handler. I suppose you wanted to track visibility change, but there's no reason to do this because visibility cannot be changed by the user.

Reply:
fixed